### ISSUE TITLE
[Bug] Long-press empty input doesn't open context menu

### ISSUE THREAD

**Issue by ervinoro** [mozilla-mobile/fenix#16572]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
Long press empty input field (e.g. reddit login email or passwd).

### Expected behavior
Context menu opens with at least paste and autofill options.

### Actual behavior
Haptic feedback and single text selection handle appears. Autofilling or pasting is impossible.

### Device information

* Android device: Xiaomi Mi A1 Android 9
* Fenix version: 82.1.3 (Build #2015774643)

**Comment by ervinoro** [comment]
I noticed that the behaviour isn't entirely consistent, sometimes I do get a context menu with just paste and without autofill. But still other times I don't. Unfortunately, I'm not sure what influences it.

**Comment by sflorean** [comment]
Tested on latest RC build and I've got the Paste option but not autofill. The saved logins menu was displayed only after the email was inserted. 
@ervinoro do you have the same result?

**Comment by ervinoro** [comment]
Yes, testing at this time I can see that long-pressing now reliably opens the context menu with the paste option. Furthermore, now navigating to a login page sometimes triggers my password manager's overlay even without activating the password field (bitwarden). The latter behaviour is unreliable and there doesn't seem to be any way to open the overlay again once dismissed, other than to kill firefox, reopen it and navigate to the page again, but i'm not sure if that's an issue with firefox or bitwarden.
